src/lib.rs

```diff
@@ -145,6 +148,25 @@ pub struct Node {
     pub id: String,
     /// List of opaque identifiers for this node's dependencies
     pub dependencies: Vec<String>,
+
+    /// Dependencies in a structured format
```

Comment:
It might be helpful to explain the difference between this and `dependencies`.  In particular, `deps` handles renamed dependencies whereas `dependencies` does not.